akvo/rsr/tests/rest/test_recipient_country.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
```

Comment:
@zzgvh - as discussed on Skype, leaving this test as is, since it tests both things.